FlowCrypt/Controllers/Threads/MessageThread.swift

```diff
@@ -33,8 +33,13 @@ struct MessageThread: Equatable {
         labels.contains(.inbox)
     }
 
-    var isArchived: Bool {
-        !labels.contains(.inbox) && !labels.contains(.sent)
+    var shouldShowMoveToInboxButton: Bool {
+        guard let firstMessageLabels = messages.first?.labels else {
+            return false
+        }
+        // Thread is treaded as archived when labels don't contain `inbox` and first message label doesn't contain sent label
+        // https://github.com/FlowCrypt/flowcrypt-ios/pull/1769#discussion_r931874353
+        return !isInbox && !firstMessageLabels.contains(.sent)
```

Comment:
I found another case when `move to archive` button isn't visible - send message from Flowcrypt, then open it:
- Gmail app shows 'move to archive` button
- Flowcrypt doesn't show it (because of `firstMessageLabels.contains(.sent)`)

Reply:
`Move to archive` button on gmail app for this case doesn't work at all i think.

Reply:
I think in gmail app, sent emails can't be archived

Reply:
Yeah, just re-checked it - `archive` button is visible just for some sent messages, but invisible for others. And message stays in `sent` folder even after tapping on this button.
Then everything is ok here 👍